newIDE/app/src/SceneEditor/index.js

```diff
@@ -281,43 +283,27 @@ export default class SceneEditor extends React.Component<Props, State> {
 
   openObjectsList = () => {
     if (!this.editorMosaic) return;
-    if (!this.editorMosaic.openEditor('objects-list', 'end', 75, 'column')) {
-      this.setState({
-        showObjectsListInfoBar: true,
-      });
-    }
+    this.editorMosaic.openEditor('objects-list', 'end', 75, 'column');
   };
 
   openProperties = () => {
     if (!this.editorMosaic) return;
-    if (!this.editorMosaic.openEditor('properties', 'start', 25, 'column')) {
-      this.setState({
-        showPropertiesInfoBar: true,
-      });
-    }
+    this.editorMosaic.openEditor('properties', 'start', 25, 'column');
   };
 
   openObjectGroupsList = () => {
     if (!this.editorMosaic) return;
     this.editorMosaic.openEditor('object-groups-list', 'end', 75, 'column');
   };
 
-  toggleInstancesList = () => {
+  openInstancesList = () => {
     if (!this.editorMosaic) return;
-    if (!this.editorMosaic.openEditor('instances-list', 'end', 75, 'row')) {
-      this.setState({
-        showInstancesInfoBar: true,
-      });
-    }
+    this.editorMosaic.openEditor('instances-list', 'end', 75, 'row');
   };
 
-  toggleLayersList = () => {
+  openLayersList = () => {
```

Comment:
maybe more of an open question.
But should all those function open AND close the corresponding panel?
I feel that this is the default behavior in most tools nowadays. You expect something to happen when you click on an icon (and you can click again if you misclicked to make it appear again)

Reply:
Probably yes!

Reply:
Done